src/main.js

```diff
@@ -81,37 +81,43 @@ function displayRandomHomePage() {
   displayRandomTitle();
   displayRandomTagline1();
   displayRandomTagline2();
+  currentCover = new Cover(htmlImage.src, htmlTitle.innerText, htmlTagline1.innerText, htmlTagline2.innerText);
+  console.log(currentCover);
 }
 //
 
 //ITERATION 2 FORM PAGE & MENU UPDATE
-
-function makeNewCover() {
+function clearNewCoverInputs() {
+  coverField.value = "";
+  titleField.value = "";
+  taglineField1.value = "";
+  taglineField2.value = "";
+}
```

Comment:
Nice! Always good to have these reusable functions handy so they can be used anytime you need them without repeating code.